shub/tool.py

```diff
@@ -23,7 +24,14 @@
 @click.group(help=HELP, short_help=SHORT_HELP, epilog=EPILOG)
 @click.version_option(shub.__version__)
 def cli():
-    pass
+    update_url = update_available()
+    if update_url:
+        click.echo(
```

Comment:
I'd keep it to one line:

    A new shub version is available. Update via pip or get it at http://github.com///